.golangci.yml

```diff
@@ -1,27 +1,24 @@
 linters:
```

Comment:
here, using the same config as in the operator. The removed modules were anyway disabled in golangci-lint